python-threatexchange/threatexchange/cli/fetch.py

```diff
@@ -295,6 +301,10 @@ def consume_descriptors(dq: collections.deque) -> int:
                 continue
             dataset.store_cache(signal_type)
             print(f"{signal_name}: {counts[signal_name]}")
+        if errors:
```

Comment:
Is it worth (optionally) logging the first N-IDs that created the errors?

Reply:
Potentially, but the place I would want that is in a logging file or a verbose option, both of which seemed out of scope for now.